README.md

```diff
@@ -98,6 +98,12 @@ $this->jws->setPayload(array(
 ));
 ```
 
+# Using a custom encoder
+
+If, for some reason, you need to encode the token in a different way, you can
+inject any implementation of `Namshi\JOSE\Base64\Encoder` in a `JWS` instance.
+Likewise, `JWS::load()` accepts such an implementation as a second argument.
+
```

Comment:
hoy man! everything looks fine to me! One last thing could you change the heading here in the doc to use an h2 (##)? then squash the commits and im ready to merge! :)